nova_vm/src/engine/bytecode/bytecode_compiler/class_definition_evaluation.rs

```diff
@@ -23,9 +23,11 @@ use crate::{
 use ahash::{AHashMap, AHashSet};
 use oxc_ast::{
     ast::{self, MethodDefinitionKind},
-    syntax_directed_operations::{BoundNames, PrivateBoundIdentifiers, PropName},
+    // syntax_directed_operations::{BoundNames, },
```

Comment:
```suggestion
```
**Question**: Is this not needed anymore?

Reply:
nope, MB forgot to remove it